BUILD

```diff
@@ -59,11 +59,16 @@ genrule(
     set -- $(SRCS)
     chmod 755 $$1
     chmod 755 $$2
-    chmod 755 dist/*.objdump
     cp $$1 dist/envoy.aar
     cp $$2 dist/envoy-pom.xml
     shift 2
-    cp $$@ dist/
+
+    mkdir -p dist/symbols
+    if [[ -f "$$@" ]]; then
```

Comment:
for multiple archs this is multiple files so this part is invalid bash I think? `if [[ -f "foo.symbols bar.symbols" ]]`